tests/tests/tier0/bluechi-is-online-system/test_bluechi_is_online_system.py

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright Contributors to the Eclipse BlueChi project
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+import logging
+from typing import Dict
+
+from bluechi_test.config import BluechiAgentConfig, BluechiControllerConfig
+from bluechi_test.machine import BluechiAgentMachine, BluechiControllerMachine
+from bluechi_test.test import BluechiTest
+
+LOGGER = logging.getLogger(__name__)
+
+AGENT_ONE = "agent-one"
+AGENT_TWO = "agent-two"
+
+
+def exec(ctrl: BluechiControllerMachine, nodes: Dict[str, BluechiAgentMachine]):
+
+    agent_one = nodes[AGENT_ONE]
+    agent_two = nodes[AGENT_TWO]
+
+    # Test 1: Both agents and controller are online
+    LOGGER.debug("Testing with both agents and controller online.")
+    result, output = ctrl.exec_run("bluechi-is-online system")
+    assert result == 0, "Expected successful execution of bluechi-is-online system"
+    assert not output, "Expected no output when all agents are online"
```

Comment:
NIT: 
I think you can skip checking the `output`. The important part is the return code, i.e. `result`. The output is only for the human, the result will be used to automate and is thus the significant part.

Reply:
Done